frontend/tests/robot/RF-tests/linkpage/TestSuite.robot

```diff
@@ -0,0 +1,25 @@
+*** Settings ***
+Documentation    Tests for the linkpage of the app.
+Resource    ../../RF-keywords/CommonFunctions.resource
+Resource    suiteKeywords.resource
+
+Suite Setup
+...        Run Keywords
+...        Open And Configure Browser
+...        AND    Register Keyword To Run On Failure    Nothing
+Suite Teardown    Close Browser
+
+Test Setup    Go To Frontpage
+Test Tags    linkpage
+
+
+*** Test Cases ***
+User can navigate to the linkpage
+    [Documentation]    User can navigate to the linkpage and click on the link
+    Go To Links Page
+    Click On Link
```

Comment:
tää vois olla erilline testicase et pystyy klikkaa linkkii, ja test setup kandeis olla Go To Links Page kosk täs testataa Linkkisivuu eikä etusivuu